spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -55,26 +55,6 @@ def foo
     expect(src).to reek_of(:NestedIterators)
   end
 
-  it 'should report nested iterators only once per method' do
-    src = <<-EOS
-      def bad(fred)
-        @fred.each {|item| item.each {|part| @joe.send} }
-        @jim.each {|ting| ting.each {|piece| @hal.send} }
-      end
-    EOS
-    expect(src).to reek_of(:NestedIterators)
-  end
-
-  it 'reports nested iterators only once per method even if levels are different' do
-    src = <<-EOS
-      def bad(fred)
-        @fred.each {|item| item.each {|part| part.foo} }
-        @jim.each {|ting| ting.each {|piece| piece.each {|atom| atom.foo } } }
-      end
-    EOS
-    expect(src).to reek_of(:NestedIterators)
```

Comment:
Those specs were completely useless in their current form since

>> expect(src).to reek_of(:NestedIterators)

is so generic that it does not fullfill the promise of the example description.